plugins/tags/plugin.cpp

```diff
@@ -492,9 +493,9 @@ namespace golos { namespace plugins { namespace tags {
                     if (citr == cidx.end()) {
                         return false;
                     }
-                    
-                    // query.reset_start_comment();
+                    query.reset_start_comment();
                     itr = idx.iterator_to(*citr);
+                    itr++;
```

Comment:
Change this to `++itr` - it works faster, then `itr++`

Reply:
Fixed that :hammer:. Thank you for the tip. Learned something new for myself